README.md

```diff
@@ -28,14 +29,21 @@ return [
 
 ## Using
 
-First all we need to set up the folder where the spec is be storing.
-This is the base folder relative for which we will structure the specification files.
+First of all, it's necessary to set up the folders where the spec is stored and where the generated doc must be placed.
+
+The api documentation we write is all inside `config_folder`.
+
+The generated html file is going to be inside `doc_folder`.
+
 ```yaml
 swagger:
-    config_folder: '%kernel.project_dir%/docs/api/'
+  apis:
+    frontend:
```

Comment:
this should be an option if you have multiply api docs. but previous way should also be supported. because backwards compatibility will be broken.
so in the result it should support old and new implementation.

Reply:
Yes, you are 100% right that this is the BC break. Although, as for me, maintenance of the current config will be a real headache. Taking into account that migration to the new approach is really simple:

```diff
swagger:
-  config_folder: '%kernel.project_dir%/docs/api/'
+  apis:
+    frontend:
+      config_folder: '%kernel.project_dir%/docs/api/'
+      doc_folder: '%kernel.project_dir%/public/api/'
```

I would suggest releasing a new major version that breaks this compatibility and writing an UPGRADE guide to point out the desired changes.

In any case, if you wish, I could implement the feature w/o BC support and you could add this support afterwards, if needed.